Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1-alpine as build
+
+# Build binary
+COPY . /atumd
+WORKDIR /atumd
+RUN go build -a -o "/bin/atumd" .
+
+# Start building the final image
+FROM scratch
+
+# Copy binary from build stage
+COPY --from=build /bin/atumd /bin/atumd
```

Comment:
Can you make a non-root user to run the application with?

https://github.com/privacybydesign/irmago/blob/02e8186ff9b08afa2151becee2cb7044425be0d7/Dockerfile#L24